central/deployment/datastoretest/datastore_sac_test.go

```diff
@@ -48,11 +43,8 @@ type deploymentDatastoreSACSuite struct {
 	suite.Suite
 
 	// Elements for bleve+rocksdb mode
-	engine   *rocksdb.RocksDB
-	index    bleve.Index
-	dacky    *dackbox.DackBox
-	keyFence dackboxConcurrency.KeyFence
-	indexQ   queue.WaitableQueue
+	engine *rocksdb.RocksDB
```

Comment:
remove these?

Reply:
Have a follow-up to refactor `DackboxTestDataStore` interface. I will address your comment in it.